policyengine_us/variables/gov/states/de/tax/income/credits/personal/de_personal_credit.py

```diff
@@ -11,8 +11,6 @@ class de_personal_credit(Variable):
     defined_for = StateCode.DE
 
     def formula(tax_unit, period, parameters):
-        p = parameters(
-            period
-        ).gov.states.de.tax.income.credits.personal_credits
-        exemptions = tax_unit("exemptions", period)
-        return p.personal * exemptions
+        p = parameters(period).gov.states.de.tax.income.credits
+        num_exemptions = tax_unit("exemptions_count", period)
```

Comment:
We try to avoid `num` since it can be ambiguous between `count` and `index`
```suggestion
        exemptions = tax_unit("exemptions_count", period)
```

Reply:
It cannot be confusing in these cases because it it clearly a local copy of the global `exemptions_count` variable.

Reply:
Using `exemptions` **is** confusing because it is the same as the new global variable `exclusions`, which (like deductions) represents the dollar amount of exemptions.

Reply:
can you use `exemptions_count` then? i agree it's obvious in this case because it's a short formula, but we're aiming for consistency and it could affect interpretation elsewhere.

Reply:
OK, I'll change this even though, as you agree, there is no possibility for confusion here.   By the way, you goal of "consistency" in code style, variable naming, etc., is a pipe dream.   The volunteer army is all over the map in coding style.